Mind-Benders/Project/css/style.css

```diff
@@ -0,0 +1,843 @@
+ 
+    .contact-address{
+        height: 480px;
+    }
+    .contact-sub-div{
+        margin:10px ;
+    }
+}
+footer{
+    background-color: #1c1c1c;
+    padding: 0px !important;
+    }
+    
+    .greytxt
+    {
+        color: #bbbbbb;
+        font-family: 'Montserrat', sans-serif;
+        font-size: 15px;
+        font-weight: 400;
+        line-height: 27px;
+    }
+    
+    .copyright{
+    background-color: #363636;
+    width: 100% !important;
+    
+    }
+    footer h3, footer h6,footer a{
+        font-family: 'Roboto', sans-serif;
+    }
+    footer h6{
+        text-transform: capitalize;
+        font-weight: bold;
+        font-size: 16px;
+    }
+    .greyborder{
+       
+        border-bottom:  1px solid gray;
+        padding-bottom: 50px;
+        margin-bottom: 25px;
+        
+    }
+    footer h3{
+        font-size: 21px;
+        font-weight: bold;
+        margin-bottom: 16px;
+    }
+    .txtsize{
+        font-size: 13px;
+    }
+    span a{
+    text-decoration: none;
+    color: white;
+    }
+    span a:hover{
+        color: teal;
+        text-decoration: none;
+    }
+    .news img{
+    width: 80px;
+    height: 80px;
+    border-radius: 50%;
+    margin-right: 20px;
+    float: left;
+    }
+    
+    .notification{
+      position: absolute;
+     background-color: #47c9e5;
+     font-size: 13px;
+     padding: 0 5px;
+    border-radius: 20px;
+    color: white;
+    z-index: 15;
+    left: 75px !important;
+    }
+    
+    .middle {
+        transition: .5s ease;
+        opacity: 0;
+        position: absolute;
+        width: 80px;
+        height: 80px;
+        border-radius: 50%;
+        float: left;
+        display: flex;
+      }
+      
+      .img:hover{
+        opacity: 0.3;
+        
+      }
+      .imgcontainer:hover .middle {
+            opacity: 0.5;
+            background-color: teal;
+      }
+    .middle img{
+        width: 25px;
+        height: 25px;
+        margin: auto;
+    }
+    .help-sec{
+      padding-top: 60px;
+      padding-bottom: 100px;
+      background-image: url(../imgs/help-background.jpg);
+      
+      background-repeat: no-repeat;
+      background-size: cover;
+    }
+    
+    .help-sec h2{
+      font-size: 30px;
+      font-weight: 700;
+      font-family: 'Roboto', sans-serif;
+    
+    }
+    .help-sec h6{
+      font-size: 12px;
+      font-weight: 400;
+      color: #363636;
+      font-family: 'Montserrat', sans-serif;
+      text-transform: uppercase;
+    }
+    .services{
+      border-radius: 10px;
+      font-family: 'Montserrat', sans-serif;
+      background-color: white;
+    }
+    
+    .services h2{
+    font-size: 18px;
+    margin-top: 30px;
+    }
+    .services p{
+      color: #363636;
+    font-size: 12px;
+    line-height: 1.7;
+    margin: 0px 0px 15px 0px;
+    display: block;
+    box-sizing: border-box;
+    max-width: 300px;
+    }
+    .sec4 {
+      width: 100%;
+    
+    }
+    .conta {
+      margin-top: 8%;
```

Comment:
it is better to use pixels rather than % with margins and padding cause % may go wrong in different screen sizes